k8s/postgresql/manifest/application.yaml.template

```diff
@@ -0,0 +1,59 @@
+    marketplace.cloud.google.com/deploy-info: '{partner_id: "google", product_id: "postgresql", partner_name: "Google Click To Deploy"}'
+  labels:
+    app.kubernetes.io/name: "$APP_INSTANCE_NAME"
+spec:
+  type: PostgreSQL
+  version: '9.6.10'
+  description: |-
+    PostgreSQL is an open source object-relational database with an emphasis on extensibility and standards-compliance.
+
+    # Support
+    Google does not offer support for this solution. However, community support is available on
+    [Stack Overflow](https://stackoverflow.com/questions/tagged/postgresql/).
+    Additional community support is available on [community forums](https://www.postgresql.org/list/).
+  info:
+  - name: Service endpoint
+    value: TODO
+  maintainers:
+  - name: Google Click to Deploy
+    url: https://cloud.google.com/solutions/#click-to-deploy
+  links:
+  - description: 'User Guide: Google Click to Deploy PostgreSQL'
+    url: https://console.cloud.google.com/code/develop/browse/google-click-to-deploy/master/k8s/postgresql?project=k8s-marketplace-eap
+  - description: 'PostgreSQL: Getting Started'
+    url: https://github.com/GoogleCloudPlatform/postgresql-docker/blob/master/9.6/README.md
```

Comment:
Should the link be https://github.com/GoogleCloudPlatform/postgresql-docker/tree/master/9/9.6/kubernetes-app?